src/components/header/index.jsx

```diff
@@ -35,7 +36,9 @@ export default function Header() {
 			<div class={style.outer}>
 				<div class={style.inner}>
 					<Nav class={style.nav} routes={config.nav} current={url} />
-					<Search />
+					<Suspense fallback={<span>Search</span>}>
+						<Search />
+					</Suspense>
```

Comment:
Will already be using `/compat` as their search is a React component, I don't think a spoofed suspense boundary would do much good here?

As for the fallback, shouldn't ever be shown to my knowledge so probably doesn't matter. Just went with text to avoid too much shifting in dev & throttled connections more than anything.